react/features/notifications/middleware.web.js

```diff
@@ -1,40 +0,0 @@
-/* @flow */
-
-import { CONFERENCE_JOINED } from '../base/conference';
-import { MiddlewareRegistry } from '../base/redux';
-import { openSettingsDialog, SETTINGS_TABS } from '../settings';
-
-import {
-    showNotification
-} from './actions';
-import { NOTIFICATION_TIMEOUT_TYPE } from './constants';
-
-import './middleware.any';
-
-/**
- * Middleware that captures actions to display notifications.
- *
- * @param {Store} store - The redux store.
- * @returns {Function}
- */
-MiddlewareRegistry.register(store => next => action => {
-    switch (action.type) {
-    case CONFERENCE_JOINED: {
-        const { dispatch, getState } = store;
-        const { disableSelfView } = getState()['features/base/settings'];
-
-        if (disableSelfView) {
-            dispatch(showNotification({
```

Comment:
I don't think removing this is a good idea. In the case that the config to disableSelfView is set to true (so the self-view is hidden, but the user can show it by going to settings) we have a discoverability issue

Reply:
You see the notification when the self view is not hidden, which is part of the report. And you see it on every join ...